feathr-example/static_features.ipynb

```diff
@@ -0,0 +1,1632 @@
+      "Requirement already satisfied: soupsieve>1.2 in /Users/dasorbit/projects/merlion-feature-store-scenario/.venv/lib/python3.8/site-packages (from beautifulsoup4->google>=3.0.0->feathr) (2.3.2.post1)\n",
+      "Requirement already satisfied: pycparser in /Users/dasorbit/projects/merlion-feature-store-scenario/.venv/lib/python3.8/site-packages (from cffi>=1.12->cryptography>=2.5->azure-identity->feathr) (2.21)\n",
+      "Requirement already satisfied: pyasn1<0.5.0,>=0.4.6 in /Users/dasorbit/projects/merlion-feature-store-scenario/.venv/lib/python3.8/site-packages (from pyasn1-modules>=0.2.1->google-auth<3.0.0dev,>=1.16.0->google-api-python-client>=2.41.0->feathr) (0.4.8)\n",
+      "\u001b[33mWARNING: There was an error checking the latest version of pip.\u001b[0m\u001b[33m\n",
+      "\u001b[0mNote: you may need to restart the kernel to use updated packages.\n"
+     ]
+    }
+   ],
+   "source": [
+    "%pip install -U feathr pandavro scikit-learn"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prerequisite: Configure the required environment\n",
+    "\n",
+    "In the first step (Provision cloud resources), you should have provisioned all the required cloud resources. If you use Feathr CLI to create a workspace, you should have a folder with a file called `feathr_config.yaml` in it with all the required configurations. Otherwise, update the configuration below.\n",
+    "\n",
+    "The code below will write this configuration string to a temporary location and load it to Feathr. Please still refer to [feathr_config.yaml](https://github.com/linkedin/feathr/blob/main/feathr_project/feathrcli/data/feathr_user_workspace/feathr_config.yaml) and use that as the source of truth. It should also have more explanations on the meaning of each variable."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import tempfile\n",
+    "yaml_config = \"\"\"\n",
+    "# Please refer to https://github.com/linkedin/feathr/blob/main/feathr_project/feathrcli/data/feathr_user_workspace/feathr_config.yaml for explanations on the meaning of each field.\n",
+    "api_version: 1\n",
+    "project_config:\n",
+    "  project_name: 'feathr_home_credit'\n",
+    "  required_environment_variables:\n",
+    "    - 'REDIS_PASSWORD'\n",
+    "    - 'AZURE_CLIENT_ID'\n",
+    "    - 'AZURE_TENANT_ID'\n",
+    "    - 'AZURE_CLIENT_SECRET'\n",
+    "offline_store:\n",
+    "  adls:\n",
+    "    adls_enabled: tru\n",
+    "  wasb:\n",
+    "    wasb_enabled: true\n",
+    "  s3:\n",
+    "    s3_enabled: false\n",
+    "    s3_endpoint: 's3.amazonaws.com'\n",
+    "  jdbc:\n",
+    "    jdbc_enabled: false\n",
+    "    jdbc_database: 'feathrtestdb'\n",
+    "    jdbc_table: 'feathrtesttable'\n",
+    "  snowflake:\n",
+    "    url: \"dqllago-ol19457.snowflakecomputing.com\"\n",
```

Comment:
Probably not needed, if so, please remove the name.

Reply:
Removed snowflake's configuration.